.circleci/config.yml

```diff
@@ -1,12 +1,34 @@
 ---
 common-steps:
+  - &install_poetry
+    run:
+      name: Install Poetry
+      command: |
+        set -e
+        source /etc/os-release
+        if [[ "$VERSION_CODENAME" == "bullseye" ]]; then
+            # Install Poetry via PyPI
+            apt-get update && apt-get install --yes --no-install-recommends python3-pip
+            pip install poetry==1.6.1
```

Comment:
(Do we want poetry 1.7.0 now that it's available?)

Reply:
Let's stay at 1.6.1 for now since that's what we're using elsewhere and we can do a coordinated bump later. Also it's a bit weird, on bullseye we're using 1.6.1, but on bookworm, the Debian package is at 1.3.2. So we might need to reconcile that too, but we can leave that for a bit later...